NEWS.md

```diff
@@ -3,6 +3,14 @@ Hit tab to unselect buttons and scroll through the text using UP/DOWN/PG UP/PG D
 When you choose 'I have read this' the message will not appear on startup anymore.
 All announcements will be stored in /opt/openhabian/docs/NEWSLOG for you to lookup.
 
+## June 17, 2020
+### removed support for PINE A64(+) and older Linux distributions
+openhabian-config will now issue a warning if you start on unsupported hardware or OS releases.
+See [README](README.md) for supported HW and OS.
+In short, PINE A64 is no longer supported and OS releases other than the current `stable` and the previous one are deprecated.
+That's Debian / Raspbian 10 (buster) and 9 (stretch) as well as Ubuntu 20.04 (focal) and 18.04 (bionic).
+Note: Running on any of these may still work or not.
```

Comment:
not sure what the note targets at? the "supported ones" or the "older" ones.
maybe just remove....

Reply:
_Eiertanz, weil ich Ubuntu noch nicht als supported reinschreiben wollte._